src/server/api/controllers/orders.controller.js

```diff
@@ -77,7 +78,18 @@ const getOrderByUserId = async (userid) => {
 };
 
 const storeNewOrder = async (data) => {
-  await knex('orders').insert(data);
+  await knex('orders')
+    .insert({ user_id: data.user_id })
+    .returning('id')
+    .then((id) => {
+      data.items.forEach((item) => {
+        knex('order_items').insert({
```

Comment:
When I tested nothing was added tto order_items table only orders.
Itried to add await here and it worked.
```
data.items.forEach(async (item) => {
          await knex('order_items').insert({
            order_id: id[0],
            product_id: item.product_id,
            quantity: item.quantity,
          });
        });
```

Reply:
Ok thanks @Jul-S i will update it 👍